gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -134,6 +135,21 @@ class ComposePlugin : Plugin<Project> {
                 }
             }
         }
+
+        checkAndWarnAboutUsingJsPlugin(project)
+    }
+
+    private fun checkAndWarnAboutUsingJsPlugin(project: Project) {
+        val msg = "Project has kotlin(\"js\") plugin applied.\n" +
+                "It's necessary to apply kotlin(\"multiplatform\") plugin " +
```

Comment:
The warning is quite long.
What do you think about `'org.jetbrains.compose' plugin is not compatible with 'org.jetbrains.kotlin.js' plugin (use 'org.jetbrains.kotlin.multiplatform' instead)`?

Reply:
fixed